proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -220,7 +220,7 @@ message UnbondingDelegation {
   string validator_address = 2 [(cosmos_proto.scalar) = "cosmos.ValidatorAddressString"];
```

Comment:
Possibly the comment above should be updated, as it is now not necessarily bech32-encoded.